packages/picasso/src/QuillEditorView/styles.ts

```diff
@@ -20,10 +21,70 @@ const margins = {
   },
 }
 
+const outlinedBullet = `url("data:image/svg+xml,%3Csvg fill='none' xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16'%3E%3Cpath fill-rule='evenodd' clip-rule='evenodd' d='M8 9c.55228 0 1-.44772 1-1s-.44772-1-1-1-1 .44772-1 1 .44772 1 1 1Zm0 1c1.10457 0 2-.89543 2-2s-.89543-2-2-2-2 .89543-2 2 .89543 2 2 2Z' fill='%23455065'/%3E%3C/svg%3E")`
+const bullet = `url("data:image/svg+xml,%3Csvg fill='none' xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16'%3E%3Ccircle cx='8' cy='8' r='2' fill='%23455065'/%3E%3C/svg%3E");`
+
+const orderedContent = (indent: number) => {
+  const decimalContent = [3, 6, 9]
+  const lowerRomanContent = [2, 5, 8]
+  const lowerAlphaContent = [1, 4, 7]
+
+  if (decimalContent.includes(indent)) {
+    return `counter(list-${indent}, decimal) "."`
+  }
+
+  if (lowerRomanContent.includes(indent)) {
+    return `counter(list-${indent}, lower-roman) "."`
+  }
+
+  if (lowerAlphaContent.includes(indent)) {
+    return `counter(list-${indent}, lower-alpha) "."`
+  }
+}
+
+const unorderedContent = (indent: number) => {
+  if (indent % 2 === 0) {
+    return bullet
+  }
+
+  return outlinedBullet
+}
+
+const indentStyles = [1, 2, 3, 4, 5, 6, 7, 8, 9].reduce(
```

Comment:
```suggestion
const indentStyles = [1, 2, 3, 4, 5].reduce(
```

I think this should only allow up to 5, not 9 (as per ticket description). Not sure if we need to allow more (as it can be intentionally limited to 5), but I'd clear this point out with the design team if we want to have up to 9.

Reply:
@mkrl ✅